File: python/h00.inheritance/Student.py
#!/usr/bin/env python
from Person import Person    # include Person.py

# class declaration
class Student(Person):
  ###### GETTER/SETTER ANTI-PATTERN ######
  def get_gpa (self):
    return self._gpa

  def set_gpa (self, gpa):
    self._gpa = gpa

File: java/h00.inheritance/Student.java
package person_space;

public class Student extends Person
{
  /******* FIELDS *******/
  private Double gpa;

  /******* MUTATORS (SETTERS) *******/
  public void   setGPA(Double gpa) { this.gpa = gpa; }

  /******* ACCESSORS (GETTERS) *******/
  public Double getGPA()           { return gpa; }
}
